### PR TITLE
added flags to unix ip

### PR DESCRIPTION
**Pull request by dre1902**

### CODE REVIEW

File: src/ifcfg/parser.py [hunk 1]
```diff
@@ -349,6 +349,8 @@ def default_interface(self):
         """
         return self._default_interface()
 
+    def alter(self, interfaces):
+        return interfaces
```
Comment (benjaoming) [inline src/ifcfg/parser.py:353]: Is this new method intentional?

**Review by benjaoming — CHANGES_REQUESTED** [review]
LGTM, although it could be supported a bit better with test assertions. I'm not sure what the new method is about?

File: src/ifcfg/parser.py (hunk 1 above)
Comment (dre1902) [reply src/ifcfg/parser.py:353]: It is. It is there to maintain the consistency from ifconfig. Without it all of the flags would be lower case as opposed to the LinuxParser which returns them all in the original upper case.

Comment (benjaoming) [reply src/ifcfg/parser.py:353]: Understood, thanks for elaborating :+1:

**Review by benjaoming — APPROVED** [review]
LGTM!

**Pull request merged**